src/org/opendatakit/briefcase/operations/Export.java

```diff
@@ -75,43 +75,32 @@
       Arrays.asList(PEM_FILE, EXCLUDE_MEDIA, OVERWRITE, START, END)
   );
 
-  public static void export(String storageDir, String formid, Path exportPath, String baseFilename, boolean includeMediaFiles, boolean overwriteFiles, Optional<LocalDate> startDate, Optional<LocalDate> endDate, Optional<Path> pemFile) {
+  public static void export(String storageDir, String formid, Path exportPath, String baseFilename, boolean includeMediaFiles, boolean overwriteFiles, Optional<LocalDate> startDate, Optional<LocalDate> endDate, Optional<Path> maybePemFile) {
     CliEventsCompanion.attach(log);
     bootCache(storageDir);
     Optional<BriefcaseFormDefinition> maybeFormDefinition = FileSystemUtils.getBriefcaseFormList().stream()
         .filter(form -> form.getFormId().equals(formid))
         .findFirst();
 
-    if (!maybeFormDefinition.isPresent())
-      throw new FormNotFoundException(formid);
-
-    BriefcaseFormDefinition formDefinition = maybeFormDefinition.get();
+    BriefcaseFormDefinition formDefinition = maybeFormDefinition.orElseThrow(() -> new FormNotFoundException(formid));
 
     if (formDefinition.isFileEncryptedForm() || formDefinition.isFieldEncryptedForm()) {
-      if (!pemFile.isPresent())
-        throw new BriefcaseException("Missing pem file configuration");
-
-      if (!Files.exists(pemFile.get()))
-        throw new BriefcaseException("Missing pem file configuration");
+      Path pemFile = maybePemFile
+          .filter(file -> !Files.exists(file))
```

Comment:
Ooh, nice! (The filter)

Reply:
In Spanish, I'd say it's a "monad-a" ("monada" means cute)

Reply:
Hey, there’s `notExists`!

Reply:
Now that I say it out loud, the word for monad is "mónada" and for cute (adj) "monada", which only have different accents (first syllable tonic and unstressed respectively)